subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/ProcessFixture.groovy

```diff
@@ -65,6 +65,27 @@ class ProcessFixture {
         return bash("ps -o pid,ppid,args -p ${pids.join(' -p ')}").split("\\n")
     }
 
+    void waitForFinish() {
```

Comment:
Could you document that this is waiting for the underlying process specified by pid to exit?